src/Data/Widget/Adapters/CellPosition.elm

```diff
@@ -0,0 +1,20 @@
+module Data.Widget.Adapters.CellPosition exposing (CellPosition, cellPositionDecoder)
+
+import Json.Decode as Decode exposing (Decoder, index, int, map2)
+
+
+type alias RowNumber =
+    Int
+
+
+type alias ColumnNumber =
+    Int
+
+
+type alias CellPosition =
+    ( RowNumber, ColumnNumber )
+
+
+cellPositionDecoder : Decoder CellPosition
```

Comment:
This is a bit nit picky but I tended to just call the method "decoder" if it was the main decoder wrapped in a module. Then it just reads `CellPosition.decoder`. Sometimes I've shared many decoders in a module tho hence them being more specifically named.

Reply:
Makes sense 👍